ndb/src/google/cloud/ndb/blobstore.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, *args, **kwargs):
 
 
 class BlobInfo:
+    __slots__ = ()
+
```

Comment:
Weird -- are all these empty-slots, only-stub-methods classes placeholders for later work?

Reply:
Indeed. I'm happy to remove them if you like. I just wanted to remind my future self, e.g. with a failing test, if they were absent. 

ASIDE: In cases where I used `__slots__`, I've typically [implemented `__dict__`][1] for users that want introspection from the console. WDYT?

[1]: https://github.com/dhermes/bezier/blob/f99db20312bb4ba7e5943195020a8ced4be9457b/src/bezier/curve.py#L147-L148

Reply:
@dhermes Don't remove them:  I was just confused that the PR was touching only things which can't (yet) be used.

Re a property for `__dict__`:  shouldn't `help(thing)` do the right thing without it?  Exposing it but not allowing people to munge its contents still breaks expectations.

Reply:
> Exposing it but not allowing people to munge its contents still breaks expectations.

Yes I agree.